### PR TITLE
1 ➡️ 2

### PR DESCRIPTION
**Pull request by chapulina**
Also added Focal CI.

### CODE REVIEW

File: dartsim/src/JointFeatures.cc [hunk 1]
```diff
@@ -89,7 +89,12 @@ void JointFeatures::SetJointAcceleration(
 void JointFeatures::SetJointForce(
     const Identity &_id, const std::size_t _dof, const double _value)
 {
-  this->ReferenceInterface<JointInfo>(_id)->joint->setForce(_dof, _value);
+  auto joint = this->ReferenceInterface<JointInfo>(_id)->joint;
+  if (joint->getActuatorType() != dart::dynamics::Joint::FORCE)
+  {
+    joint->setActuatorType(dart::dynamics::Joint::FORCE);
```
Comment (scpeters) [inline dartsim/src/JointFeatures.cc:95]: @azeey I guess codecov didn't run for #74, but it's now complaining that the patch coverage is too low. This is the only line without coverage. Did you expect this line to be covered by the current tests?

Comment (azeey) [reply dartsim/src/JointFeatures.cc:95]: Ah, no, I don't think we have a test that uses both SERVO and FORCE actuator types. I can add one to ign-physics1, but I'm not sure I can get to it this week.

Comment (scpeters) [reply dartsim/src/JointFeatures.cc:95]: it's ok; I was just checking

**Comment by scpeters** [comment]
@osrf-jenkins run tests please

**Comment by scpeters** [comment]
I'm going to manually squash the commits that are exclusive to this branch before merging

**Comment by scpeters** [comment]
I couldn't figure out how to squash into a merge commit (3fb287d), so I just squashed the commits on top of that

**Comment by scpeters** [comment]
windows build had a flaky failure; I'm ignoring it and will merge

**Pull request merged**